examples/demo_app/bin/bundle

```diff
@@ -1,3 +1,105 @@
 #!/usr/bin/env ruby
```

Comment:
The changes in this file come from https://github.com/rails/rails/commit/920c9195e0e27077af003973e45d0146a30e51b4